src/ExtranetBundle/LoginByTransferToken/TransferTokenService.php

```diff
@@ -0,0 +1,130 @@
+<?php
```

Comment:
Naming: What does this transfer? It sounds as if it transfers a token?

Reply:
It's a service that handles tokens in order to log extranet users in - in order to transfer between two domains for example. If this was a service to transfer tokens I would expect it to be named `TokenTransferService`.

Reply:
So the main purpose is login?

LoginByTokenService?

"Transfer" would be an edge case / implementation detail?

Reply:
ok

Reply:
:heavy_check_mark: Renamed